doc/source/configuration.rst

```diff
@@ -0,0 +1,270 @@
+        # or user_domain_id: '<123456_DOMAIN_ID>'
+        auth_url: 'https://iam.eu-de.otc.t-systems.com:443/v3'
+      interface: 'public'
+      identity_api_version: 3 # !Important
+      ak: '<AK_VALUE>' # AK/SK pair for access to OBS
+      sk: '<SK_VALUE>'
+
+.. note::
+   The name ``otc`` is self-defined and can be changed to any value.
+
+AK/SK values required for access to some services (i.e. OBS) can
+be either configured as shown above in the clouds.yaml/secure.yaml, or
+they can be automatically retrieved from the S3_ACCESS_KEY_ID and
+S3_SECRET_ACCESS_KEY.
+
+Test your connection
+^^^^^^^^^^^^^^^^^^^^
+
+If you followed the `installation advices <index>`_ for your specific
+operating system, you can use the following command to test the basic
+functionality.
+
+.. code-block:: bash
+
+    $ openstack --os-cloud otc flavor list
+
+
+Configuration of a Second Project
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Additional connections to other Openstack-clouds or -projects can be added
+to the file as shown below:
+
+**clouds.yaml**
+
+.. code-block:: yaml
+
+  clouds:
+    otcfirstproject:
+      profile: otc
+      auth:
+        username: '<USER_NAME>'
+        password: '<PASSWORD>'
+        project_name: '<eu-de_project>'
+        # or project_id: '<123456_PROJECT_ID>'
+        user_domain_name: 'OTC00000000001000000xxx'
+        # or user_domain_id: '<123456_DOMAIN_ID>'
+        auth_url: 'https://iam.eu-de.otc.t-systems.com:443/v3'
+      interface: 'public'
+      identity_api_version: 3 # !Important
+      ak: '<AK_VALUE>' # AK/SK pair for access to OBS
+      sk: '<SK_VALUE>'
+    otcsecondproject:
+      profile: otc
+      auth:
+       username: '<USER_NAME>'
+        password: '<PASSWORD>'
+        project_name: '<eu-de_project2>'
+        # or project_id: '<123456_PROJECT_ID2>'
+        user_domain_name: 'OTC00000000001000000xxx'
+        # or user_domain_id: '<123456_DOMAIN_ID2>'
+        auth_url: 'https://iam.eu-de.otc.t-systems.com:443/v3'
+      interface: 'public'
+      identity_api_version: 3 # !Important
+      ak: '<AK_VALUE>' # AK/SK pair for access to OBS
+      sk: '<SK_VALUE>'
+
+Splitting the credentials in clouds.yaml and secure.yaml
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In some scenarios a split of security credentials from the configuration file
+is necessary. The optional file ``secure.yaml`` can be used to store the
+secret which is left out from ``clouds.yaml``:
+
+**clouds.yaml**
+
+.. code-block:: yaml
+
+  clouds:
+    otc:
+      profile: otc
+      auth:
+        username: '<USER_NAME>'
+        project_name: '<eu-de_project>'
+        # or project_id: '<123456_PROJECT_ID>'
+        user_domain_name: 'OTC00000000001000000xxx'
+        # or user_domain_id: '<123456_DOMAIN_ID>'
+        auth_url: 'https://iam.eu-de.otc.t-systems.com:443/v3'
+      interface: 'public'
+      identity_api_version: 3 # !Important
+      ak: '<AK_VALUE>' # AK/SK pair for access to OBS
```

Comment:
would you please also place sk into the secrets.yaml section here? We want to show proper case from security pov

Reply:
done